.github/CODEOWNERS

```
@@ -0,0 +1,57 @@
+#####################################################
+#
+# List of approvers for OpenTelemetry Go Contrib
+#
+#####################################################
+#
+# Learn about membership in OpenTelemetry community:
+#  https://github.com/open-telemetry/community/blob/main/community-membership.md
+#
+#
+# Learn about CODEOWNERS file format:
+# https://help.github.com/en/articles/about-code-owners
+#
+# NOTE: Lines should be entered in the following format:
+# <component_path_relative_from_project_root>/<min_1_space><owner_1><space><owner_2><space>..<owner_n>
+# instrumentation/net/http/otelhttp/         @open-telemetry/collector-go-approvers @madvikinggod @mralias
+# Path separator and minimum of 1 space between component path and owners is
+# important for validation steps
+# 
+
+* @open-telemetry/go-approvers
+
+detectors/aws/                                                          @open-telemetry/go-approvers
+detectors/gcp/                                                          @open-telemetry/go-approvers
+
+instrumentation/github.com/astaxie/beego/otelbeego/                     @open-telemetry/go-approvers
+instrumentation/github.com/aws/aws-lambda-go/otellambda/                @open-telemetry/go-approvers
+instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/                   @open-telemetry/go-approvers
+instrumentation/github.com/bradfitz/gomemcache/memcache/otelmemcache/   @open-telemetry/go-approvers
+instrumentation/github.com/emicklei/go-restful/otelrestful/             @open-telemetry/go-approvers
+instrumentation/github.com/gin-gonic/gin/otelgin/                       @open-telemetry/go-approvers
+instrumentation/github.com/go-kit/kit/otelkit/                          @open-telemetry/go-approvers
+instrumentation/github.com/gocql/gocql/otelgocql/                       @open-telemetry/go-approvers
+instrumentation/github.com/gorilla/mux/otelmux/                         @open-telemetry/go-approvers
+instrumentation/github.com/labstack/echo/otelecho/                      @open-telemetry/go-approvers
+instrumentation/github.com/Shopify/sarama/otelsarama/                   @open-telemetry/go-approvers
```

From https://github.com/open-telemetry/opentelemetry-go-contrib/issues/2834#issuecomment-1331095757

```suggestion
instrumentation/github.com/Shopify/sarama/otelsarama/                   @open-telemetry/go-approvers @pellared 
```